lib/workers/repository/update/branch/index.ts

```diff
@@ -526,6 +526,13 @@ export async function processBranch(
         config.forcePr = true;
         config.branchAutomergeFailureMessage = mergeStatus;
       }
+      if (mergeStatus === 'off schedule') {
+        logger.warn(
+          'Branch cannot automerge now because it is off schedule - raising a PR instead'
+        );
+        config.forcePr = true;
+        config.branchAutomergeFailureMessage = mergeStatus;
+      }
```

Comment:
Why raising a pr? this would cause PR most of the time.

@rarkins shouldn't we simply skip this update then?

Reply:
Yes, we should just skip it and try again next time. Don't create a PR because of automerge schedule

Reply:
Removed `config.forcePr`. not sure if that what was meant. 
doing so will skip pr creation when `automergeType="branch"`.
but not for type `pr`, in the later a pr will be raised.

for  `automergeType="branch"` we will get this in the dashboard:
perhaps we could drop the "branch" in  `Pending Branch Automerge` to cover both grounds.
and also change the note, as we are not really waiting for status checks. for one, im running with `"ignoreTests": true` in my  cfg.


WDYT?
____
This issue provides visibility into Renovate updates and their statuses. [Learn more](https://docs.renovatebot.com/key-concepts/dashboard/)

## Repository problems

These problems occurred while renovating this repository.

 - WARN: Branch cannot automerge now because automergeSchedule is off schedule - skipping

## Pending Branch Automerge

These updates await pending status checks before automerging. Click on a checkbox to abort the branch automerge, and create a PR instead.

 - [ ] <!-- approvePr-branch=renovate/ubuntu-22.x -->Update dependency ubuntu to v22

## Detected dependencies

<details><summary>dockerfile</summary>
<blockquote>

<details><summary>Dockerfile</summary>

 - `ubuntu focal`

</details>

</blockquote>
</details>